Provider.react.js

```diff
@@ -1,6 +1,47 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import LocaleContext from './Context';
+import {getHostName} from 'virtual-tour-viewer/src/utils/UIHelper';
+
+async function isLocalHost() {
+    let hostname = !window.origin ? window.location.origin : window.origin;
+    return hostname.includes('localhost');
+}
+
+async function getLabelsFromHavrApi(locale) {
+    let taxonomyName = 'spaces';
+    // defaulting to en-US in case locale isn't present
+    locale = locale ? locale : 'en-US';
+    let tardisResponse;
+    if (await isLocalHost()) {
+        tardisResponse = await fetch(
+            `http://localhost:8080/havr-api/v2/taxonomies/${taxonomyName}/${locale}`
```

Comment:
Note sure why hostname is hardcoded here but it's recommended to pick it from either a properties file or env variables